src/Packages/Passport/Runtime/Scripts/Private/Core/BrowserCommunicationsManager.cs

```diff
@@ -95,13 +96,23 @@ private void CallFunction(string requestId, string fxName, string data = null)
 
             // Call the function on the JS side
             string js = $"callFunction(\"{requestJson}\")";
-            Debug.Log($"{TAG} Call {fxName} (request ID: {requestId}, js: {js})");
+
+            if (fxName != PassportAnalytics.TRACK)
+            {
+                string dataString = data != null ? $": {data}" : "";
+                PassportLogger.Info($"{TAG} Call {fxName} (request ID: {requestId}){dataString}");

```

Comment:
It looks to me {TAG} will be duplicated in the log message since it is already a part of PassportLogger.Log method?

Reply:
The `TAG` in this file differs from the tag in `BrowserCommunicationsManager`.
In this file the TAG is `[Browser Communications Manager]`, so when this line is printed it will say:
`[Immutable] [Browser Communications Manager] Call ...`